Kickstarter-iOS/Features/Messages/Controller/MessagesViewController.swift

```diff
@@ -39,6 +43,12 @@ internal final class MessagesViewController: UITableViewController {
 
     _ = self.replyBarButtonItem
       |> UIBarButtonItem.lens.title %~ { _ in Strings.general_navigation_buttons_reply() }
+
+    if let banner = messageBannerViewController?.view {
+      NSLayoutConstraint.activate([
+        banner.widthAnchor.constraint(equalTo: self.view.widthAnchor)
+      ])
```

Comment:
I'm surprised to see that you need to manipulate `messageBannerViewController?.view` here. Two questions:

1. Why is this extra constraint necessary for this PR but not for other/older uses of `MessageBannerViewController`?
2. Can this be moved into a method on `MessageBannerViewController`? Something like `messageBannerViewController.fixTheBannerWidth(to: self.view.widthAnchor)` would be more self-documenting.

Reply:
So for some reason, that I haven't quite figured out yet, when this banner is added to. UITableViewController a right constraint of 1000 is added. If you look in `MessageBannerViewController` you'll see that it should be constrained to the right and left constraint of its parent view, but something is overwriting that. 
I like your idea of moving this to `MessageBannerViewController ` so that it can be more self-documenting. Without knowing the root cause I'm not sure how to approach that though.

Reply:
Very weird! (But intriguing). Have you tried the constraints/view debugger? Sometimes if you print the description of the constraint it'll tell you where it came from.

Reply:
I couldn't see why the tableviewcontroller was adding this extra trailing constraint to the banner but, I was able to get around this by setting the banner's widthAnchor rather than using leftAnchor + rightAnchor. 

I tested the other areas of the app where we use this banner and everything still looks good there.